README.md

```diff
@@ -61,19 +54,14 @@ Now you can move it into the `PATH` to use it from any directory:
 mv ./secrethub /usr/local/bin/
 ```
 
-### Use a package manager
-
-We plan to distribute secrethub via package managers ([#27](https://github.com/secrethub/secrethub-cli/issues/27)).
-Please feel free to join the discussion and let us know what package manager you are using.
-
-## Test your installation
+### Test your installation
 
 Verify your installation works by running the following command:
 ```sh
 secrethub --version
 ```
 
-## Install auto-completion
+### Install auto-completion
```

Comment:
I like your idea of moving this to the website and just having a reference to that in the README.